dkg/spectest/tests/resharing.go

```diff
@@ -0,0 +1,73 @@
+package tests
+
+import (
+	"github.com/bloxapp/ssv-spec/dkg"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// ResharingHappyFlow tests a simple (dummy) resharing flow, the difference between this and keygen happy flow is
+// resharing doesn't sign deposit data
+func ResharingHappyFlow() *MsgProcessingSpecTest {
```

Comment:
wasn't added to Alltests so not sure it's running

Reply:
Enabled in https://github.com/bloxapp/ssv-spec/pull/82/commits/5a7d054b437990a8434fa20d5d3dc322c046ea1b